src/components/Lists/ExpenseList.jsx

```diff
@@ -15,13 +16,32 @@ const ExpensesList = ({expData}) => {
   let expenseData = expData;
   console.log('prop', expData)
   const [expenses, setExpenses] = React.useState([]);
+  const [filterCategory, setFilterCategory] = React.useState('');
+  const [filterResults, setFilterResults] = React.useState([]);
+  const [noResultsFound, setNoResultsFound] = React.useState(false);
 
-  React.useEffect(() => {
+  const fetchDataFromCache = () => {
     const storedExpenses = JSON.parse(localStorage.getItem('expenses')) || expenseData;
     setExpenses(storedExpenses);
-    console.log('inside expense list', storedExpenses, 'el', expenses)
+    setFilterResults(storedExpenses);
+    console.log('inside expense list', storedExpenses)
+  }
+
+  // runs only on the first render
+  React.useEffect(() => {
```

Comment:
Instead of using React.useEffect(), you can utilize destructuring when importing React.